R/downloadMehta2024.R

```
@@ -23,7 +22,8 @@
 
 downloadMehta2024 <- function(subtype = "GMIA") {
 
-  aeiURL <- "https://zenodo.org/records/7809342"
+  aeiURL <- "https://zenodo.org/records/14219723"
```

So you did change the raw data, yes? Will someone, who has downloaded the old data before, get the update? Or will madrat silently continue to read in the same outdated data?